все что смогла сделать

Astridsie opened the pull request:
проблемы:
1. не получается дать два класса одновременно. программа видит только первый
2. не получается убрать подчеркивание у ссылок в header с помощью text-decoration: none; во всех остальных частях сайта ссылки убираются
3. не получается в футере покрасить ссылки в белый цвет.
4. не получилось сделать фон вокруг картинок

Astridsie committed:
все что смогла сделать

проблемы:
1. не получается дать два класса одновременно. программа видит только первый
2. не получается убрать подчеркивание у ссылок в header с помощью text-decoration: none; во всех остальных частях сайта ссылки убираются
3. не получается в футере покрасить ссылки в белый цвет.
4. не получилось сделать фон вокруг картинок

Yan1993-Dev commented on index.html:
```diff
@@ -4,10 +4,14 @@
-    <header>
+    <header class="container">
         <a href="#"><img src="./img/icon/DLC.svg" alt="dlc"></a>
        <u>
```
Почему у тебя тег u, а не ul

Yan1993-Dev commented on catalog.html:
```diff
@@ -0,0 +1,83 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="css/common.css">
```
Путь до файла не правильно прописан, поставь в начале ./

Yan1993-Dev commented on contact.html:
```diff
@@ -0,0 +1,94 @@
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,700;1,400&family=Open+Sans:ital,wght@0,400;0,700;1,700&display=swap" rel="stylesheet">
+
+    <title>Window Modal</title>
```
Названия должны совпадать с смыслом страницы

Yan1993-Dev commented on contact.html:
```diff
@@ -0,0 +1,94 @@
+                <input id="check" type="checkbox">
+                <button class="blackbutton" type="button" >Submit</button>
+            </div>
+            </form>
```
Текст с макета не вставила

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,96 @@
+    font-style: normal;
+}
+
+.blackbutton {
```
Именования класса не универсальное. Если поменяется кнопки цвет? Пропиши название класса btn, button

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,96 @@
+.blackbutton {
+    font-family: 'Open Sans';
+    padding: 11px 40px;
+font-style: normal;
+font-weight: 700;
+font-size: 16px;
+line-height: 22px;
+align-items: center;
+text-align: center;
+text-transform: uppercase;
+
+
+color: #FFFFFF;
+background: #000000
```
Соблюдай отступы и убирай лишние пробелы

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,96 @@
+    font-style: italic;
+}
+
+fieldset {
+    width: 1400px;
+height: 254px;
+left: 0px;
+top: 301px;
+
+background: rgba(125, 151, 218, 0.2);
+}
```
Не задавай ширину, высоту, отступы с помощью left: 0px;
top: 301px;
Есть специальные свойства для отступов. padding, margin

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,96 @@
+    font-size: 26px;
+}
+
+form {
+max-width: max-content;
+height: 542px;
+left: 110px;
+top: 301px;
+background: #E5EAF8
```
Указывая всем формам определенный размер, ты не сможешь добавить дополнительно другую форму. max-width: max-content; устанавливаем для кнопок

Astridsie committed:
.

пыталась сделать хедер и футер.
1. не получилось сделать звездный рейтинг по гайду.
2. я не знаю как сделать так чтобы серая линяя не двигала списки

Astridsie committed:
Merge branch 'dl-academy-edu:main' into main
